### PR TITLE
Add Bitmagnent

### PR DESCRIPTION
**Pull request by DrMxrcy**

### CODE REVIEW

File: apps/bitmagnet/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+      - --keys=http_server
+      - --keys=queue_server
+      # disable the next line to run without DHT crawler
+      - --keys=dht_crawler
+    container_name: bitmagnet
+    environment:
+      - POSTGRES_HOST=bitmagnet-db
+      - POSTGRES_NAME=bitmagnet
+      - POSTGRES_USER=bitmagnet
+      - POSTGRES_PASSWORD=postgres
```
Comment (meienberger) [inline apps/bitmagnet/docker-compose.yml:19]: Should be a variable

**Review by meienberger — CHANGES_REQUESTED** [review]

**Comment by DrMxrcy** [comment]
Can we get a re-review?

File: apps/bitmagnet/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  bitmagnet:
+    image: ghcr.io/bitmagnet-io/bitmagnet:0.1.1
+    restart: unless-stopped
+    command:
+      - worker
+      - run
+      - --keys=http_server
+      - --keys=queue_server
+      # disable the next line to run without DHT crawler
+      - --keys=dht_crawler
```
Comment (steveiliop56) [inline apps/bitmagnet/docker-compose.yml:13]: Will that work?

**Comment by steveiliop56** [comment]
@DrMxrcy so we merge?

**Review by meienberger — DISMISSED** [review]
LGTM (except the logo)

**Review by meienberger — APPROVED** [review]

**Pull request merged**